### DISCUSSION TITLE
Translation to

### DISCUSSION THREAD

**Discussion by henriquecavet** [Leantime/leantime#992] [Q&A] [answered]
Hello guys.

I need to translate the field effort and priority. Where i find this fields? I thanks any to help me!

Sincerely,

**Comment by marcelfolaron** [comment, marked as answer]
Hi, the translation files are in
resources/languages/

Are you creating a new translation or changing existing ones?

**Reply by henriquecavet** [reply]
Hi I'm just changing the existing one. needed to change the options, which are still in english, i need to change to PT_BR, is this the best option?

**Comment by joaobrazuna** [comment]
Olá,
A versão pt_pt está atualizada para a versão 2.2.7.
Podes atualizar a PT_BR a partir dessa.
:-)

**Reply by henriquecavet** [reply]
Opa, Blz.... vou ajustar... muito obrigado!
